_includes/header.html

```diff
@@ -1,23 +1,78 @@
-<nav class="navbar navbar-expand-md navbar-light bg-light noprint">
+<nav class="navbar navbar-expand-md noprint">
+
   <a class="navbar-brand" href="/">
-    <img src="https://static.opensuse.org/chameleon/dist/images/logo/logo-white.svg" class="mr-2" alt="Logo">
-    <span>News</span>
+    <svg class="icon icon-2x mr-2">
+      <use xlink:href="#opensuse"></use>
+    </svg>
+    <span class="navbar-title">News</span>
   </a>
+
   <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#site-menu">
-    <span class="navbar-toggler-icon"></span>
+    <svg class="icon">
+      <use xlink:href="#menu-line"></use>
+    </svg>
   </button>
+
   <div class="collapse navbar-collapse" id="site-menu">
-    <div class="nav navbar-nav ml-auto">
-      <div class="nav-item">
-        <a class="nav-link" href="{{ "/feed.xml" | prepend: site.baseurl }}">
-          <span class="fa fa-rss"></span>
-          <span class="d-md-none">RSS Feed</span>
+    <ul class="nav navbar-nav mr-md-3">
+      <!-- Categories Dropdown -->
+      <li class="nav-item dropdown">
+        <a class="nav-link dropdown-toggle" href="#" id="cat-menu-link" role="button" data-toggle="dropdown"
+          aria-haspopup="true" aria-expanded="false">
+          <svg class="icon">
+            <use xlink:href="#newspaper-line"></use>
+          </svg>
+          Categories
         </a>
+        <div class="dropdown-menu" aria-labelledby="cat-menu-link">
+          {% for category in site.categories %}
+          {% capture category_name %}{{ category | first }}{% endcapture %}
+          <a class="dropdown-item" href="/category/{{ category_name | slugify }}">
+            {{ category_name }}
+          </a>
+          {% endfor %}
+        </div>
+      </li>
+
+      <li class="nav-item">
+        <a class="nav-link" href="https://github.com/openSUSE/news-o-o#how-to-submit-a-new-post">
+          <svg class="icon">
+            <use xlink:href="#quill-pen-line"></use>
+          </svg>
+          New Post
+        </a>
+      </li>
+
+      <li class="nav-item">
+        <a class="nav-link" href="{{ '/feed.xml' | prepend: site.baseurl }}">
+          <svg class="icon">
+            <use xlink:href="#rss-line"></use>
+          </svg>
+          RSS Feed
+        </a>
+      </li>
+    </ul>
+
+    <!-- DuckDuckGo Site Search -->
+    <form id="ddg-form" class="form-inline" action="https://duckduckgo.com/">
```

Comment:
Align right for breakpoints larger than md

Reply:
Done.